features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -24,19 +24,10 @@ Feature: cluster monitoring related upgrade check
     Given I switch to cluster admin pseudo user
     And I use the "openshift-monitoring" project
     Given all pods in the project are ready
-    Then the step should succeed
-    When I run the :get client command with:
-      | resource      | clusteroperator |
-      | resource_name | monitoring      |
-    And evaluation of `@result[:stdout].split(/\n/)[1].split(/\s+/)[4]` is stored in the :degreaded_status clipboard
-    Then the expression should be true> cb.degreaded_status == "False"
+    And the expression should be true> cluster_operator('monitoring').condition(type: 'Degraded')['status'] == "False"
```

Comment:
Have you execute the code? I didn't find the definition of method condition

Reply:
```
    def conditions(user: nil, cached: true, quiet: false)
      rr = raw_resource(user: user, cached: cached, quiet: quiet)
      rr.dig('status', 'conditions')
    end

    # returns a hash simliar to
    # {"lastTransitionTime"=>2019-03-23 07:33:34 UTC, "status"=>"False", "type"=>"Progressing"}
    def condition(type: nil, user: nil, cached: true, quiet: false)
      res = self.conditions(user: user, cached: cached, quiet: quiet).select { |c| c['type'] == type }
      res.first
    end
```

Reply:
thanks, I find it.